File: app/src/main/java/pl/moozo/apicalc/data/remote/ApiUtils.java
package pl.moozo.apicalc.data.remote;

public class ApiUtils {

    private ApiUtils() {}
    public static final String BASE_URL = "http://192.168.0.100:8080/";

    public static APIService getAPIService() {

        return RetrofitClient.getClient(BASE_URL).create(APIService.class);
    }
}

File: app/src/main/java/pl/moozo/apicalc/data/remote/ApiUtils.kt
package pl.moozo.apicalc.data.remote

object ApiUtils {
    val BASE_URL = "http://192.168.0.100:8080/"

    val apiService: APIService
        get() = RetrofitClient.getClient(BASE_URL)!!.create(APIService::class.java)
}
